packages/app/src/app/pages/Sandbox/Editor/Workspace/Advertisement/Advertisement.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { useScript } from 'app/hooks';
+import { Container } from './elements';
+
+export const Advertisement = () => {
+  useScript(`https://codefund.app/properties/24/funder.js`);
```

Comment:
Nice and clean!

Reply:
Yup! Now we can reuse useScript anywhere we need to load an outside script on component mount.